apps/portal/src/app/app-routes.ts

```diff
@@ -45,10 +45,12 @@ export const appRoutes: Routes = [
                                     },
                                     {
                                         path: 'report',
-                                        loadChildren: () =>
+                                        loadComponent: () =>
                                             import(
-                                                './report-component/report-routes'
-                                            ).then((m) => m.reportRoutes),
+                                                '@sol/admin/class-printouts'
+                                            ).then(
+                                                (m) => m.ClassPrintoutsComponent
+                                            ),
```

Comment:
Directly loads the `ClassPrintoutsComponent` on the `/admin/report` route.
Previously, there was a `reportRoutes` array that was used.